README.md

```diff
@@ -1,65 +1,63 @@
 # Poandy
 
-Simple Python wrapper for Oanda (https://developer.oanda.com/rest-live-v20/introduction/).
+A simple Python wrapper for [OANDA v20 API](https://developer.oanda.com/rest-live-v20/introduction/).
 
 ## Setup
 
-1. Create neccessary files.
+1. Rename the file `secrets.example.json` to `secrets.json`, then change the value of `token` to your own OANDA v20 API key.
 
-   ```
-   # navigate to Poandy directory
-   cd poandy
-   # create a file called "secrets.json". It should contain your oanda api key. E.g.
-   {
-       "token": "7851c27f3bb2bc0f39g3c6d6a3c6b42509e-4015643c1a6ca4651b0c6bd836bc8b8"
-   }
-   ```
+2. Set up a virtual environment and install dependencies with Anaconda.
 
-2. Setup virtual environment, activate and install neccessary dependencies. For Windows and Linux, the requirements files are at `requirements/win-64.txt` and `requirements/linux-64.txt` respectively.
+```
+# create virtual environment and install dependencies in environment.yml
+conda env create --file environment.yml
+# activate virtual environment
+conda activate poandy
+# when done
+conda deactivate
+```
+
+## Development
```

Comment:
I think we should include the part where we need to set up a secret with our github account in order to be able to push to the repo.

Reply:
for me, as long as readme has instructions for users

Reply:
@kaneishere if CI removal is approved (in the other PR), there is no need to set up the secret anymore - else I'll include it in